### PR TITLE
Es-843418-Add notes in PowerPoint animation to images

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/843418">Add notes in PowerPoint Animation to Images</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add notes in PowerPoint Animation to Images</br>
Add youtube link in Word to PDF and Word to Image
</td>
</tr>

</table>

Regards,
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Word-document/Compare Word documents.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Comparing two Word documents allows you to identify the changes between two vers
 
 N> 1. DocIO performs word level comparison while comparing two Word documents.In this scenario, if a single character in a word is changed, the entire word will be highlighted as changed.
 N> 2. Comparing two Word documents supported in DOCX format only.
+N> 3. Converting PowerPoint slides to images based on animation effects is not supported on the UWP platform.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Word-document/Compare Word documents.md:18]: You added note in Compare Word documents. Please add in PowerPoint UG of corresponding topic.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/word-to-pdf.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ documentation: UG
 
 The Word document files are converted as a PDF document with a few lines of code by using the Essential DocIO. It works perfectly when you create an input Word document from scratch or load an existing Word document and easily converted into PDF. 
 
+To quickly start converting a Word document to PDF, please check out this video:
```
Comment (Chrispineagunja) [inline File-Formats/DocIO/word-to-pdf.md:15]: To quickly start converting a Word document to a PDF, please check out this video:

File: File-Formats/Presentation/Presentation-to-image.md [hunk 3]
```diff
@@ -205,7 +205,8 @@ The .NET PowerPoint Library (Presentation) allows you to convert PowerPoint slid
 For instance, if a slide includes bulleted paragraphs, each having entrance animation effects, the Presentation library converts every bulleted paragraph into a separate image.
 
 N> 1. Only entrance animation effects are considered for generating separate images. Other animation effects and non-animated elements will be converted into images within the first image itself.
-N> 2. Converting PowerPoint animations to images is not supported in the UWP platform. 
+N> 2. Converting PowerPoint animations to images is not supported in the UWP platform.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-image.md:208]: N> 2. The UWP platform does not support the conversion of PowerPoint animations to images.

File: File-Formats/Presentation/Presentation-to-image.md [hunk 4]
```diff
@@ -205,7 +205,8 @@ The .NET PowerPoint Library (Presentation) allows you to convert PowerPoint slid
...
+N> 3. Converting PowerPoint slides to images based on animation effects is not supported on the UWP platform.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-image.md:209]: N> 3. The UWP platform does not support the conversion of PowerPoint slides to images based on animation effects.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: File-Formats/Presentation/Presentation-to-image.md [hunk 5]
```diff
@@ -205,7 +205,8 @@ The .NET PowerPoint Library (Presentation) allows you to convert PowerPoint slid
 For instance, if a slide includes bulleted paragraphs, each having entrance animation effects, the Presentation library converts every bulleted paragraph into a separate image.
 
 N> 1. Only entrance animation effects are considered for generating separate images. Other animation effects and non-animated elements will be converted into images within the first image itself.
-N> 2. Converting PowerPoint animations to images is not supported in the UWP platform. 
+N> 2. The UWP platform does not support the conversion of PowerPoint animations to images.
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/Presentation-to-image.md:208]: Please remove duplicate note for same requirement

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Word-document/Compare Word documents.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Word-document/Compare Word documents.md]: Updated sir.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**